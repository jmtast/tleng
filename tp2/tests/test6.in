// Error de semántica: Compás más largo de lo indicado
#tempo negra 120
#compas 2/4

const oct1 = 5;

voz (0)
{
  compas
  {
    silencio(redonda);
  }
}
