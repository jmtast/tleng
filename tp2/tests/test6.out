Error:
Compás con duración (1.0) más larga que la indicada (0.5). Línea 9
